sdk/requirements.txt

```diff
@@ -1,8 +1,9 @@
+fire
 google-auth>=1.30.0,<2.0dev
 google-cloud-storage>=1.37.0
 mlflow==1.14.1
 numpy==1.17.3
 pandas==1.2.2
 python-dateutil
+requests==2.25.1
```

Comment:
The latest version (`2.26.0`) is not working correctly with `urllib3-mock`, so I have to pin the version explicitely